### PR TITLE
Update MODEL_ZOO.md

### PR DESCRIPTION
**Pull request by scott-vsi**

### CODE REVIEW

**Comment by scott-vsi** [comment]
It's just a broken link. Feel free to correct and close this PR.

File: MODEL_ZOO.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This file documents a large collection of baselines trained with Detectron, prim
 - All models were trained on the union of `coco_2014_train` and `coco_2014_valminusminival`, which is exactly equivalent to the recently defined `coco_2017_train` dataset.
 - All models were tested on the `coco_2014_minival` dataset, which is exactly equivalent to the recently defined `coco_2017_val` dataset.
 - Inference times are often expressed as "*X* + *Y*", in which *X* is time taken in reasonably well-optimized GPU code and *Y* is time taken in unoptimized CPU code. (The CPU code time could be reduced substantially with additional engineering.)
-- Inference results for boxes, masks, and keypoints ("kps") are provided in the [COCO json format](http://cocodataset.org/#format).
+- Inference results for boxes, masks, and keypoints ("kps") are provided in the [COCO json format](http://cocodataset.org/#format-data).
```
Comment (ir413) [inline MODEL_ZOO.md]: Good catch.

**Comment by ir413** [comment]
Thanks for the PR @scott-vsi. Could you please sign the CLA (see above) so that we can merge the PR?

**Comment by scott-vsi** [comment]
I work for a company so cannot sign it. I'll close the PR. Feel free to make the change yourself.

**Comment by ir413** [comment]
Thanks for letting us know.

**Pull request closed without merging**